Zilon.Core/Zilon.Core/Commands/Globe/MoveGroupCommand.cs

```diff
@@ -54,61 +54,61 @@ public bool CanExecute()
         /// <summary>Выполнение команды.</summary>
         public void Execute()
         {
-            var selectedNodeViewModel = (IGlobeNodeViewModel)_globeUiState.SelectedViewModel;
-
-            var currentNode = _player.GlobeNode;
-            var currentGlobeCell = _player.Terrain;
-            var region = _worldManager.Regions[currentGlobeCell];
-
-            if (region == selectedNodeViewModel.ParentRegion)
-            {
-                // Значит путешествие внутри одной провинции.
-
-                var neighborNodes = region.GetNext(currentNode);
-                var selectedIsNeighbor = neighborNodes.Contains(selectedNodeViewModel.Node);
-
-                if (selectedIsNeighbor)
-                {
-                    var globeNode = selectedNodeViewModel.Node;
-
-
-                    if (globeNode.Scheme.SectorLevels != null || _player.GlobeNode.IsTown)
-                    {
-                        ScoreManager?.CountPlace(globeNode);
-                    }
-
-                    _player.GlobeNode = globeNode;
-
-                    // Обновление состояния разведки узлов провинции
-                    globeNode.ObservedState = GlobeNodeObservedState.Visited;
-
-                    UpdateSurvivals();
-                }
-            }
-            else
-            {
-                Debug.Assert(selectedNodeViewModel.ParentRegion == null, "Для узла должна быть задана провинция.");
-
-                var currentTerrainNode = _player.GlobeNode;
-                var currentTerrainCell = _player.Terrain;
-                //TODO Выборку ячейки мира по узлу провиции нужно упростить.
-                var targetNeighborTerrainCell = _worldManager.Regions.Single(x => x.Value == selectedNodeViewModel.ParentRegion).Key;
-                var targetNeighborBorders = selectedNodeViewModel.ParentRegion.Nodes.OfType<GlobeRegionNode>().Where(node => node.IsBorder);
-                var transitionNodes = RegionTransitionHelper.GetNeighborBorderNodes(currentTerrainNode,
-                                                                                    currentTerrainCell,
-                                                                                    targetNeighborBorders,
-                                                                                    targetNeighborTerrainCell);
-                if (transitionNodes.Contains(selectedNodeViewModel.Node))
-                {
-                    _player.GlobeNode = selectedNodeViewModel.Node;
-                    _player.Terrain = targetNeighborTerrainCell;
-
-                    // Обновление состояния разведки узлов провинции
-                    selectedNodeViewModel.Node.ObservedState = GlobeNodeObservedState.Visited;
-
-                    UpdateSurvivals();
-                }
-            }
+            //var selectedNodeViewModel = (IGlobeNodeViewModel)_globeUiState.SelectedViewModel;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/kreghek/Zilon_Roguelike/pullRequest?prid=5260137)